app/controllers/automated_tests_controller.rb

```diff
@@ -173,6 +186,9 @@ def assignment_params
                      :max_marks, :run_on_submission, :run_on_request,
                      :halts_testing, :display_description, :display_run_status,
                      :display_marks_earned, :display_input,
-                     :display_expected_output, :display_actual_output])
+                     :display_expected_output,:display_actual_output,
+		     :_destroy],

```

Comment:
Align the elements of an array literal if they span more than one line.<br>Tab detected.